### DISCUSSION TITLE
Editing code from the mobile application

### DISCUSSION THREAD

**Discussion by Moosems** [community/community#20214] [Mobile] [answered]
I believe that it would be useful to allow people to edit files from the mobile app. This would be particularly useful for updating the personal readme files or for small fixes that you think of on the go. If there is already a way to do this O would greatly appreciate that knowledge

**Comment by elisealix22** [comment, marked as answer]
Hi @Moosems , thank you so much for the request! This is definitely on our minds and we have a tracking issue internally. It's helpful to hear that this is important to you! I can't speak to a timeline unfortunately but we are definitely thinking about it.

**Reply by FunProgramer** [reply]
This feature would be useful.

**Reply by Moosems** [reply]
What?

**Reply by Moosems** [reply]
Why not call their support service?

**Comment by hfhbd** [comment]
I really miss it too. Without editing code, I think the app is a little bit useless. 
And GitHub mobile already supports editing code! 
But unfortunately, editing code is only available when viewing the changed code in a PR. So please add a button in the code view to edit it and show the already existing UI.
